src/uikit.cs

```diff
@@ -1264,6 +1264,10 @@ interface NSLayoutManager : NSCoding {
 		[iOS (9,0)] // documented as 7.0 but missing in 8.x
 		[Export ("CGGlyphAtIndex:")]
 		ushort GetGlyph (nuint glyphIndex);
+
+		[Watch (5, 0), TV (12, 0), iOS (12, 0)]
+		[Export ("limitsLayoutForSuspiciousContents")]
+		bool LimitsLayoutForSuspiciousContents { get; set; }
```

Comment:
Replace `[Watch]` by `[NoWatch]`
It's not reported by xtro and it's not in watchOS headers files
https://github.com/xamarin/xamarin-macios/wiki/UIKit-watchOS-xcode10-beta5